devel/rapidjson-devel/Portfile

```diff
@@ -22,6 +22,8 @@ platforms                       darwin
 # original by {@Lord-Kamina gmail.com:g.litenstein}
 maintainers                     nomaintainer
 
+conflicts                       rapidjson
```

Comment:
is the recursive conflict present in the "rapidjson" Portfile?

Reply:
There is a
```
conflicts rapidjson-devel
```
line in the rapidjson Portfile, but there wasn't the
```
conflicts rapidjson
```
line in the rapidjson-devel Portfile.

I noticed it because I originally installed the rapidjson port, before realising that it was outdated. Then the rapidjson-devel failed to compile. After fixing it, it failed to activate because there were already files installed by the rapidjson port.